app/api_topologies.go

```diff
@@ -147,6 +147,13 @@ func init() {
 			Name:     "Hosts",
 			Rank:     4,
 		},
+		APITopologyDesc{
+			id:       "weave",
+			parent:   "hosts",
+			renderer: render.WeaveRenderer,
+			Name:     "Weave Net",
+			Rank:     3,
```

Comment:
I ask, because it is the same as Pods, but other sub-topologies don't seem to have a Rank at all.